packages/syft-cli/tests/version_test.py

```diff
@@ -0,0 +1,12 @@
+# third party
+from cli import app
+from typer.testing import CliRunner
+
+runner = CliRunner()
+
+
+def test_version():
```

Comment:
Flagging this in the Cool Down Period:

Currently to be able to move fast, we could shift adding github tests for the new CLI in the cool down period.

We would need to add
- A new workflow file for testing the new CLI
- Adding a tox test which automatically test the CLI


I will create a separate Github Issue for the same.


Currently, I have tested the tests manually.

Reply:
Maybe  we could add the only tox test in this PR, itself, (as it is a short change)

We could create one with name `hagrid.test.unit` like we have for syft.test.unit

Kindly have a look at tox.ini, replicate syft.test.unit for hagrid.

Reply:
We could keep the task of adding it to Github CI in the cool down period.